packages/cli/src/commands/list/command.ts

```diff
@@ -20,6 +20,15 @@ export const listCommand = {
       type: [String],
       deprecated: false,
     },
+    {
+      name: 'policy',
+      description:
+        'Preview deployment with Deployment Retention policies (e.g.: `-p KEY=value`). Can appear many times.',
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** avoid word "preview"

I think it's confusing to tell the user they can "preview" something with "preview" deployments. Does this sound better?

```suggestion
        'See deployments with provided Deployment Retention policies (e.g.: `-p KEY=value`). Can appear many times.',
```